### DISCUSSION TITLE
SparkKubernetesOperator how to delete existing spark application

### DISCUSSION THREAD

**Discussion by yevsh** [apache/airflow#16287] [General]
airflow version:  v2.0.2

 dag:

```
t1 = SparkKubernetesOperator(
    task_id='spark_pi_submit',
    namespace="dummy",
    application_file="spark.yaml",
    kubernetes_conn_id="kubernetes",
    do_xcom_push=True,
    dag=dag,
)
```

if spark application already exists in k8s cluster (under Sparkapplications) ,   execution fails on:

> airflow.exceptions.AirflowException: Exception when calling -> create_custom_object: (409)
> Reason: Conflict

> HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"sparkapplications.sparkoperator.k8s.io \"xxx\" already exists","reason":"AlreadyExists","details":{"name":"xxx","group":"sparkoperator.k8s.io","kind":"sparkapplications"},"code":409}

Is there out-of-the-box option to delete application? What is the correct way to delete

**Comment by big-loser** [comment]
did you resolve it ?
